CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 4.3.1 (Under development)
 
+### App Center 
+
+* **[Fix]** Remove CoreTelephony dependecy for maCatalyst platform. This fixed a broken SPM integration for macCatalyst when using swift 5.5
```

Comment:
Fix `Undefined symbol: OBJC_CLASS$_CTTelephonyNetworkInfo` error for Mac Catalyst platform when integrating the SDK via Swift Package Manager with Swift 5.5 and higher.